src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosJobFramework.scala

```diff
@@ -31,7 +32,7 @@ class MesosJobFramework @Inject()(
   val frameworkName = "chronos"
   private[this] val log = Logger.getLogger(getClass.getName)
   private var lastReconciliation = DateTime.now.plusSeconds(config.reconciliationInterval())
-  private var runningTasks = new HashMap[String, ChronosTask]
+  private var runningTasks = new mutable.HashMap[String, ChronosTask]
```

Comment:
Again same question with the amps in this class. What is the design choice for making them mutable?